pandas/core/config.py

```diff
@@ -140,7 +137,7 @@ def _describe_option(pat='', _print_desc=True):
     if len(keys) == 0:
         raise OptionError('No such keys(s)')
 
-    s = u('')
+    s = u''
```

Comment:
Can get rid of the u prefix now, no?

Reply:
Are we "full speed ahead" on ripping out py2?  If not, let's hold off.  I'd like to get this and #25757 merged since there are nice follow-ups.

Reply:
yes, you can remove PY2 things

Reply:
OK, just pushed.